command.h

```diff
@@ -159,8 +159,11 @@ struct cmd {
     unsigned quit : 1;      /* quit request? */
     unsigned noforward : 1; /* not need forward (example: ping) */
 
-    int slot_num; /* this command should send to which slot?
-                   * -1:the keys in this command cross different slots*/
+    /* Command destination */
+    int slot_num;    /* Command should be sent to slot
+                      * -1: multiple keys in command */
```

Comment:
Are multiple keys in same slot still OK? Just multiple keys mapped to multiple slots is -1? The old comment said cross slot...

What about the case when slot is not used; when a command is sent to explicit node? Is -1 used in this case as well?

Reply:
corrected